SPECS-EXTENDED/reaper/reaper.spec

```
@@ -0,0 +1,156 @@
+Summary: Cassandra Reaper
+Name: reaper
+Version: 3.1.1
+Release: 1%{?dist}
+License: Apache 2.0
+Vendor: Microsoft Corporation
+Distribution: Mariner
+Group: Applications/System
+
+URL: http://cassandra-reaper.io/
+Source0: https://github.com/thelastpickle/cassandra-reaper/archive/refs/tags/%{version}.tar.gz
```

The URL is good, but `rpmbuild` expects the correct file name to be the last element of the path, so you'll need to add this:

```suggestion
Source0: https://github.com/thelastpickle/cassandra-reaper/archive/refs/tags/%{version}.tar.gz#/[tarball_name]
```

---

Addressed.